sessions/src/main/proto/spine_examples/pingh/sessions/rejections.proto

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+syntax = "proto3";
+
+package spine_examples.pingh.sessions;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.pingh.spine.io";
+option java_package = "io.spine.examples.pingh.sessions.rejection";
+option java_multiple_files = false;
+
+import "spine_examples/pingh/sessions/identifiers.proto";
+import "spine_examples/pingh/github/values.proto";
+
+// The username of the authenticated account differs from
+// the one entered at the start of the authentication process.
+//
+message UsernameMismatch {
+
+    // The ID of the session.
+    SessionId id = 1 [(required) = true];
+
+    // The name of the user whose account completed the authentication.
+    spine_examples.pingh.github.Username logged_in_user = 2 [(required) = true];
+}
+
+// The user is not a member of an organization authorized to use the Pingh application.
+//
+// To function properly and display members, the organization must have
+// the Pingh application installed on GitHub.
+//
+// See: https://github.com/apps/pingh-tracker-of-github-mentions.
+//
+message OrgAccessDenied {
+
+    // The ID of the session.
+    SessionId id = 1 [(required) = true];
+
+    // Organizations that are permitted to use the Pingh application.
+    repeated spine_examples.pingh.github.Organization permitted_organization = 2;
```

Comment:
Instead of the list of permitted organizations, we should list the one to which they "wanted" to get an access, but failed.